src/database/migrations/20220805164704-create-group.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Groups', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true,
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      permissions: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        get() {
+          return this.getDataValue('permissions').split(';');
+        },
+        set(val) {
+          this.setDataValue('permissions', val.join(';'));
+        },
+      },
+      createdAt: {
```

Comment:
I think, the main problem was not adding column "userIds" here with reference to "Users" table.